cl/users/api_views.py

```diff
@@ -130,6 +130,17 @@ def test_webhook(self, request, *args, **kwargs):
                 da_dummy_curl = da_curl_template.render(
                     {"endpoint_url": webhook.url}
                 ).strip()
+            case WebhookEventType.SEARCH_ALERT:
+                da_template = loader.get_template(
+                    "includes/search_alert_webhook_dummy.txt"
+                )
+                da_dummy_content = da_template.render().strip()
+                da_curl_template = loader.get_template(
+                    "includes/search_alert_webhook_dummy_curl.txt"
+                )
+                da_dummy_curl = da_curl_template.render(
+                    {"endpoint_url": webhook.url}
+                ).strip()
```

Comment:
`da_` in this context stands for Docket Alert, doesn't it? Need some variable clean up here?

Reply:
Of course, updated to `event_` to be general.